pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -86,6 +86,30 @@ process prepareBusco {
     """
 }
 
+
+/**
+*@input path to the genome fasta
+*@output path to annotation GTF
+*@output path to genome fasta
+*/
+process linkAnnoCache {
```

Comment:
If `params.anno_cache` is set to `${params.results_dir}/anno_cache`, what does the `linkAnnoCache` process do?

Reply:
The linkAnnoCache process reuses genome annotations from the specified cache directory.

Reply:
I'm thinking about this specific case, but from our discussion outside the PR I understand that this is not how the anno_cache is expected to be used.